Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TabView/TabViewPage.xaml.cs

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.ObjectModel;
+using Microsoft.Toolkit.Uwp.UI.Controls;
+using Microsoft.Toolkit.Uwp.UI.Extensions;
+using Windows.UI.Popups;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    public sealed partial class TabViewPage : Page, IXamlRenderListener
+    {
+        public ObservableCollection<DataItem> TabItemCollection { get; } = new ObservableCollection<DataItem>();
+
+        private TabView _tabs;
+        private TabView _tabItems;
+
+        private int _counter = 1;
+
+        public TabViewPage()
+        {
+            this.InitializeComponent();
+
+            TabItemCollection.Add(new DataItem() { MyText = "First Tab", Value = 100 });
+            TabItemCollection.Add(new DataItem() { MyText = "Tab 2", Value = 200 });
+            TabItemCollection.Add(new DataItem() { MyText = "Third Tab", Value = 300 });
+            TabItemCollection.Add(new DataItem() { MyText = "Tab Plus", Value = 400 });
+
+            SampleController.Current.RegisterNewCommand("Add Upper Tab", AddUpperTabClick);
+        }
+
+        public void OnXamlRendered(FrameworkElement control)
+        {
+            control.DataContext = this;
+
+            _tabs = control.FindChildByName("Tabs") as TabView;
+            if (_tabs != null)
+            {
+                _tabs.TabDraggedOutside += Tabs_TabDraggedOutside;
```

Comment:
These never get fired since the tab can't reorder

Reply:
Right, I need to add it to the other one or keep it if I have a flag to enable it with the properties panel or something once I clean the demo up.